enhancements/split_helm_ansible_into_their_repos.md

```diff
@@ -0,0 +1,283 @@
+- [x] Test plan is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+
+## Summary
+
+Currently, the code implementation required to build the operator image used the Dockerfile of the projects built with the SDK CLI tool are in the directories [operator-sdk/internal/ansible/](https://github.com/operator-framework/operator-sdk/tree/master/internal/ansible) and [operator-sdk/internal/helm/](https://github.com/operator-framework/operator-sdk/tree/master/internal/helm).
+
+Also, their code used to scaffold the files are very are in the [operator-sdk/internal/plugins/ansible/v1](https://github.com/operator-framework/operator-sdk/tree/master/internal/plugins/ansible/v1) and [operator-sdk/internal/plugins/helm/v1](https://github.com/operator-framework/operator-sdk/tree/master/internal/plugins/helm/v1)
+
+A new release version of their images (e.g `quay.io/operator-framework/helm-operator:v1.0.0` and `quay.io/operator-framework/ansible-operator:v1.0.0`) is generated for all releases even when no changes have been made that would impact these images.
+
+All operators which are build with the tool ought to has the same based config and Makefile but has a design which allow its customizations and caveats be done on top, however, for Ansible and Helm these templates were copied from upstream(Kubebuilder) and now all these files are duplicated on the repository.   
+
+This proposal has the goal to describe how we can split Helm/Ansible projects and then, decoupled from the CLI tool as provide a design to avoid and minimize scenarios such as each type following a different approach to config and providing others defaults which would be more risk after the split by making upstream source of truth and building as its utils base. 
+
+**Note** It is a follow up of the proposal [Split SDK into multiple projects](split-sdk-into-multiple-projects.md).   
+
+## Open Questions
+
+1. Any reason for not generate the samples on the Operator-Type repos? 
+1. The templates copied from Kubebuilder were set in Kubebuilder as internal by us in the plugin phase 1 implementation. We use the plugins, and the templates are part of them, so I do not see any reason for we not be allowed to import its templates as well. However, do you see any objection to it?      
+1. The steps to reach the goal for the Operator Types projects have their own docs subdomains `ansible.operatorframework.io`, and `helm.operatorframework.io` shows have certain complexity. Have we any reason for it has not its specific proposal?
+
+## Motivation
+
+- Improve the maintainability, readability and reusability by adopting concepts such as [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns) and [Single Responsibility principle](https://en.wikipedia.org/wiki/Single-responsibility_principle). 
+- Facilitate the development process and encourage more contributions
+- Allow Lifecycle releases per type
+- Minimize the CI processing time
```

Comment:
Hi @estroz, IMO the time to process the CI has been a big problem for us. So, I agree with you that it is not a Goal. However, I think that it is a Motivation for we put it in place.